app/webpacker/scripts/cookie-banner.js

```diff
@@ -16,6 +16,10 @@ export default class CookieBanner {
       '[data-qa="cookie-banner__accept"]'
     );
 
+    this.rejectButton = this.$module.querySelector(
+      '[data-qa="cookie-banner__reject"]'
```

Comment:
This is previous code, but I don’t think it’s very clear – it suggests the data attribute is for testing, but actually it’s required to make the JS work. We have 2 options:

* Use a different data attribute, i.e. `data-accept-cookie="true"` (this is what GOV.UK Notify does)
* Use a class name, i.e. `js-cookie-banner-accept`

Whatever the choice, where we require hooks in the HTML to communicate with JS, we should not be using `data-qa-*`.